tests/installation/module_registration/register_extensions_and_modules.pm

```diff
@@ -11,12 +11,24 @@
 use base 'y2_installbase';
 use strict;
 use warnings;
-use testapi qw(save_screenshot get_required_var);
+use testapi qw(save_screenshot get_required_var get_var);
 
 sub run {
     my @scc_addons = grep($_, split(/,/, get_required_var('SCC_ADDONS')));
     $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
     save_screenshot;
+    $testapi::distri->wait_for_separate_regcode({
+            timeout => 60,
+            interval => 2,
+            message => 'Page to insert module registration code did not appear'});
+    foreach my $addon (@scc_addons) {
+        if ($addon =~ /we|ha|ltss/ && get_required_var('SCC_REGCODE_' . uc $addon)) {
+            my $regcode = get_var('SCC_REGCODE_' . uc $addon);
```

Comment:
we don't need to do this operation twice to get the setting, it is not necessary to test the infrastructure, you can remove it from the `if` condition and we assume it is there